src/js/game/LevelMVC/LevelView.js

```diff
@@ -565,11 +565,19 @@ module.exports = class LevelView {
     const blockIndex = (this.yToIndex(position[1])) + position[0];
     const block = this.actionPlaneBlocks[blockIndex];
     const animation = this.playScaledSpeed(block.animations, "activation");
+    this.controller.levelModel.actionPlane.getBlockAt(position).isEmissive = true;
```

Comment:
```suggestion
    block.isEmissive = true;
```

Reply:
🤦‍♂️ 

I thought the block there was a local copy.

Reply:
On that not, I'm not seeing the fow update when I just use block.isEmissive. Is this another bug somehow, or was my intuition about local copy vs a reference to levelModel.actionPlane's data? 🤔